VERSION

```diff
@@ -0,0 +1 @@
+v0.1.12
```

Comment:
In the current release process, we create a Github release, which triggers a Github Action and it uploads to PyPi and there we don't have to worry about this, so now this will be out of sync and release will be a two step process (updating this file and creating a release with the same tag) instead of just one.

So, I think the release Github action should probably make a commit to update this file with the user specified user tag before making a release:
https://github.com/Quansight/qhub-ops/blob/master/.github/workflows/release.yaml

cc @costrouc

Reply:
@amit we need to update setup.py anyway when creating a release. https://github.com/Quansight/qhub-ops/blob/master/setup.py#L13

Reply:
Ok, in that case this looks good to me (apart from a minor comment). We can make the CI change in an another PR, to make it one step.

Reply:
Agree